console/cypress/support/e2e.ts

```diff
@@ -18,31 +18,3 @@ import './commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
-
```

Comment:
This code was extremely flaky and only used to debug one issue. I think it makes sense to delete it.